ocs_ci/framework/conf/default_config.yaml

```diff
@@ -87,6 +87,22 @@ ENV_DATA:
   worker_replicas: 3
   skip_ocp_deployment: false
   skip_ocs_deployment: false
+  logging_version: '4.2'
+  # --------------------------------------------------------------------------
```

Comment:
I guess that the lines bellow is the badly resolved conflict after the rebase. Those were deleted when we did cleanup of our old deployment and we don't need and use the bellow values anymore so it shouldn't be there. Please delete.